docs/troubleshooting.md

```diff
@@ -80,7 +81,7 @@ If you need help debugging the CatalogSources, see [Where to go for help](#where
 Next, make sure that the `packagemanifests` were created by the `OLM package-server`. 
 
 ```
-$ oc get packagemanifests -n openshift-marketplace
```

Comment:
This is correct. It is the output that is wrong.

Reply:
@aravindhp this is what I see on my cluster:

```
$ oc get packagemanifests

NAME            CATALOG         AGE
packageserver   OLM Operators   40h
```